docs/reST/ref/sdl2_video.rst

```diff
@@ -183,6 +183,16 @@
       | :sl:`Copy a portion of the texture to the rendering target.`
       | :sg:`draw(srcrect=None, dstrect=None, angle=0, origin=None, flip_x=False, flip_y=False) -> None`
 
+   .. method:: draw_triangle
+      
+      | :sl:`Copy a triangle portion of the texture to the rendering target by vertices coordinates.`
+      | :sg:`draw_triangle(p1_xy, p2_xy, p3_xy, p1_uv=(0.0, 0.0), p2_uv=(1.0, 0.0), p3_uv=(1.0, 1.0), p1_mod=(255, 255, 255, 255), p2_mod=(255, 255, 255, 255), p3_mod=(255, 255, 255, 255)) -> None`
```

Comment:
Is the order of the p1_uv, p2_uv and p3_uv relevant? Because its not the same as on line 773 in video.pyx

Reply:
Oh, I didn't realize that. I've made a fix commit.